tests/PasswordTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+use App\Classes\Database;
+use PHPUnit\Framework\TestCase;
+
+class PasswordTest extends TestCase
+{
+
+    public static PDO $db;
+    public static $Password_Obj;
+    public static $Query;
+    public static array $id= [];
+
+    public static function setUpBeforeClass(): void
+    {
+        self::$db = Database::getInstance();
+        self::$Password_Obj = new \App\Classes\Password();
+        self::$Query = new App\Classes\Query();
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        $id_array = implode('\', \'', self::$id);
+        $delete_sql = 'DELETE FROM users WHERE user_id IN(' . '\'' . $id_array . '\')';
+        $delete_stmt = self::$db->prepare($delete_sql);
+        $delete_stmt->execute();
+    }
+
+    public function testCheckEmail()
```

Comment:
This unit test only checks the failure condition. It should also include a successful one